### PR TITLE
deportealdia.live

### LINKED ISSUE #74115: deportealdia.live
### Issue URL (Anti Adblock Script)
 [https://deportealdia.live/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fdeportealdia.live%2F)
### Comment
Username: `@BarhamBasha`
> As shown, when adguard works, the send button disappears.
> 
> Source: https://adbull.me/twwP

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> BarbBlock<br/><br/><b>Language-specific:</b><br/> Liste AR

### PR DESCRIPTION
**Pull request by wout-codes**
#74115
`"prevent-window-open"` is needed for AdGuard App.

### CODE REVIEW

**Comment by Yuki2718** [comment]
Add `deportealdia.live#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")` even though cosmetic exception alone disarms the anti-adb. Your original scriptlet is fine, the reason for this scriptlet is just to consolidate to existing solution (https://github.com/AdguardTeam/AdguardFilters/issues/56588) so that we can easily find what sites use the same class of anti-adb later.
Also add `deportealdia.live,techgeek.digital###overlay` so that user need one less click on the last page before desired contents.

**Comment by wout-codes** [comment]
> Add `deportealdia.live#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")` even though cosmetic exception alone disarms the anti-adb. Your original scriptlet is fine, the reason for this scriptlet is just to consolidate to existing solution (#56588) so that we can easily find what sites use the same class of anti-adb later.
> Also add `deportealdia.live,techgeek.digital###overlay` so that user need one less click on the last page before desired contents.

Where? General ext?

**Comment by Yuki2718** [comment]
> Where? General ext?

For scriptlet antiadblock.txt above the `! https://github.com/AdguardTeam/AdguardFilters/issues/73943` with comment line of the issue number.

The cosmetic should be specific.txt.

**Comment by wout-codes** [comment]
@Yuki2718 Also this works `deportealdia.live,adbull.me#%#//scriptlet('abort-current-inline-script', 'document.getElementById', 'adb-enabled')`

**Comment by Yuki2718** [comment]
>  Also this works `deportealdia.live,adbull.me#%#//scriptlet('abort-current-inline-script', 'document.getElementById', 'adb-enabled')`

Already commented: https://github.com/AdguardTeam/AdguardFilters/pull/74120#issuecomment-774494139
Sorry wrongly edited your comment.

**Comment by wout-codes** [comment]
> > Also this works `deportealdia.live,adbull.me#%#//scriptlet('abort-current-inline-script', 'document.getElementById', 'adb-enabled')`
> 
> Already commented: [#74120 (comment)](https://github.com/AdguardTeam/AdguardFilters/pull/74120#issuecomment-774494139)
> Sorry wrongly edited your comment.

What do you mean?

**Comment by Yuki2718** [comment]
Often there are many ways to solve an issue, but it's best start from searching for existing solution and only if there's none, use an unique solution.

**Comment by wout-codes** [comment]
> Often there are many ways to solve an issue, but it's best start from searching for existing solution and only if there's none, use an unique solution.

I agree I change the fixes by looking at the similar fixes by the Filter Devs, if I make an unique rule.

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -544,6 +544,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115 
+deportealdia.live#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:548]: We have a similar rule. Exchange the rule by adding the domain to the similar rule.

File: EnglishFilter/sections/general_extensions.txt [hunk 2]
```diff
@@ -229,8 +229,10 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115
+deportealdia.live#%#//scriptlet("prevent-window-open")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/73799
-techgeek.digital#%#//scriptlet("prevent-window-open")
+noticiasesports.live,techgeek.digital#%#//scriptlet("prevent-window-open")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:235]: No need. We have the rule in the `AdGuard Spanish`. Please check your changes before commiting to our repo.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:548]: @zloyden Which rule?

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:548]: I added the domain the rules I knew that it should work but @Yuki2718 suggested to add this rule to consolidate (make it stronger).

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:548]:
```
! https://github.com/AdguardTeam/AdguardFilters/issues/56588
key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

You should replace the task link and change the rule a bit.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/74115
deportealdia.live,key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:548]: Is it even needed? The large diffs disarms the anti-adblock

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:548]: Check.

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -254,6 +254,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+deportealdia.live,techgeek.digital###overlay
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:257]: This rule is for what?

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:257]: https://github.com/AdguardTeam/AdguardFilters/pull/74120#issuecomment-774494139

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:257]: Ok, but I can't reproduce it. @Yuki2718 can you provide a screenshot for this element?

File: EnglishFilter/sections/general_extensions.txt [hunk 4]
```diff
@@ -229,6 +229,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115
+deportealdia.live#%#//scriptlet("prevent-window-open")
```
Comment (zloyden) [inline EnglishFilter/sections/general_extensions.txt:233]: I've noticed that site is Spanish. Please move the rule to the AdGuard Spanish.

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:548]: Also move it to the AdGuard Spanish.

File: EnglishFilter/sections/antiadblock.txt [hunk 5]
```diff
@@ -1708,7 +1710,8 @@ claimtrx.club#%#//scriptlet("set-constant", "show_ads", "true")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56627
 leakite.com#%#//scriptlet("abort-on-property-write", "ai_check_block")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/56588
-key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115
+deportealdia.live,key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:1714]: Remove `deportealdia.live,` and return the original task `! https://github.com/AdguardTeam/AdguardFilters/issues/56588`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
There is a leftover that should be blocked.
`deportealdia.live#?##secondary section.widget_text:has(> h3.widget-title:contains(Publicidad))`

**Comment by wout-codes** [comment]
> There is a leftover that should be blocked.
> `deportealdia.live#?##secondary section.widget_text:has(> h3.widget-title:contains(Publicidad))`

Spanish General ext?

**Comment by zloyden** [comment]
> Spanish General ext?

No. Spanish -> css_extended.txt

**Comment by zloyden** [comment]
Now everything is fine. Let's wait @Yuki2718 response or if you can provide a screenshot - this will also count.

**Comment by Yuki2718** [comment]
@zloyden This is only seen in just before the destination page:

<details>

deportealdia

</details>

No harm with other rules but still hiding this leads to one less click.

**Comment by zloyden** [comment]
> This is only seen in just before the destination page:

Strange that I still don't have it, but ok. Thanks)

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
@zloyden Will the Pop-up Blocker UserScript be updated anytime soon? This is actually the Pop-Up Blocker's job, to be honest. It's useless especially in sites which injects redirects just right when you go the site (unsolicited redirects).

**Pull request merged**